opentelemetry-user-events-logs/examples/basic.rs

```diff
@@ -24,15 +24,10 @@ fn main() {
     let layer = layer::OpenTelemetryTracingBridge::new(&logger_provider);
     tracing_subscriber::registry().with(layer).init();
 
-    // event_name is now passed as an attribute, but once https://github.com/tokio-rs/tracing/issues/1426
-    // is done, it can be passed with name:"my-event-name", so it'll be available as metadata for
-    // fast filtering.
-    // event_id is also passed as an attribute now, there is nothing in metadata where a
-    // numeric id can be stored.
     error!(
-        event_name = "my-event-name",
-        event_id = 20,
         user_name = "otel user",
-        user_email = "otel@opentelemetry.io"
+        //event_name = "my-event-name",
```

Comment:
@cijothomas what's the expected way (end goal) of specifying event name?

Reply:
It is supposed to be `error!( name: "myevent", ...)` ref - https://github.com/tokio-rs/tracing/blob/b8c45ccd067a6aebdd9257c5edda6bc90bcf9028/tracing/src/lib.rs#L247

Have updated the example accordingly. Thanks for pointing out.